### PR TITLE
Removing extra "d" after close() method in SessionTest.java

### PR DESCRIPTION
**Pull request by rajendraarora16**

### CODE REVIEW

**Comment by rajendraarora16** [comment]
I signed it!

**Comment by rajendraarora16** [comment]
@googlebot please check CLA

File: tensorflow/java/src/test/java/org/tensorflow/SessionTest.java [hunk 1]
```diff
@@ -151,7 +151,7 @@ public void failOnUseAfterClose() {
       s.close();
       try {
         s.runner().run();
-        fail("methods on a close()d session should fail");
+        fail("methods on a close() session should fail");
```
Comment (asimshankar) [inline tensorflow/java/src/test/java/org/tensorflow/SessionTest.java]: The `d` there was intentional, to read as the verb `closed`. However, I see it isn't particularly standard.

I don't feel too strongly about the change, but if you'd like to make one perhaps phrase it something like:

"methods on a session should fail after close() is called"
?

**Review by asimshankar — CHANGES_REQUESTED** [review]

File: tensorflow/java/src/test/java/org/tensorflow/SessionTest.java (hunk 1 above)
Comment (rajendraarora16) [reply tensorflow/java/src/test/java/org/tensorflow/SessionTest.java]: Thanks! Updated.

**Review by asimshankar — APPROVED** [review]

**Comment by rajendraarora16** [comment]
@asimshankar I am struggling with tests for this change.

**Comment by KingRando95** [comment]
Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by yifeif** [comment]
@tensorflow-jenkins test this please

**Pull request merged**